homeassistant/components/tuya/cover.py

```diff
@@ -319,15 +319,19 @@ def open_cover(self, **kwargs: Any) -> None:
             commands.append(
                 {
                     "code": self.entity_description.set_position,
-                    "value": 0,
+                    "value": round(
+                        self._set_position_type.remap_value_from(
+                            100, 0, 100, reverse=True
```

Comment:
I'm a little confused here. Won't the `reverse` flag cause this to return the min here, or 0?
Perhaps I misunderstood, but I thought you concurred that open is 100.

Reply:
First 100 = open, which is Home Assistant spec.
Home Assistant has a range for open/close from 0, 100, which are the second and third parameters.

The reverse parameter means the scale in Tuya is reversed (so in most cases, this will send a 0 to Tuya).

Reply:
Doesn't this Tuya documentation for product category `mc` imply that 0 would be closed?:
<img width="574" alt="image" src="https://user-images.githubusercontent.com/4990645/145434663-95d3d59b-ce62-43c5-ab93-d4528cfdeef9.png">

Reply:
I submitted a request on the Tuya site that they clarify the docs for `cl`
I see they fixed the docs from my last request about the garage door (to clarify the mapping of the boolean status to open/closed), so hopefully they'll do the same here.

Reply:
> Doesn't this Tuya documentation for product category mc imply that 0 would be closed?:

`mc` is not supported, so not a concern right now.